src/main/java/Greet.java

```diff
@@ -0,0 +1,79 @@
+public class Greet {
+    private static final int MAX_ARRAY_LIMIT = 100;
+    protected static Task[] list = new Task[MAX_ARRAY_LIMIT];
+    private static int tasksAdded = 0;
```

Comment:
Perhaps the naming of this variable could be more meaningful since it is an int. tasksAdded sounds a bit more like a void method.